routes/clients.rb

```diff
@@ -130,7 +130,8 @@ def self.registered(app)
       app.delete '/clients/:id/?' do
         # delete a client
         client = Client[id: params[:id]]
-        client.remove unless client.name == 'default'
+        default_project = Project.find(name: 'default').id
+        client.remove default_project unless client.name == 'default'
```

Comment:
Don't forget to remove the argument if you remove that parameter from the method as suggested